jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/AppengineApiConfiguration.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.apphosting.vmruntime.jetty9;
+
+import org.eclipse.jetty.util.URIUtil;
+import org.eclipse.jetty.util.resource.Resource;
+import org.eclipse.jetty.webapp.AbstractConfiguration;
+import org.eclipse.jetty.webapp.WebAppClassLoader;
+import org.eclipse.jetty.webapp.WebAppContext;
+
+import java.util.Arrays;
+import java.util.logging.Level;
+
+public class AppengineApiConfiguration extends AbstractConfiguration {
+
+  // Hide the server implementation classes from the webapplication
+  // TODO Regularly review this list; or automate generation; or change to whitelist
+  private static final String[] SERVER_CLASSES = {
+    "com.google.",
+    "javax.cache.",
+    "org.apache.commons.",
+    "org.apache.geronimo.",
+    "org.apache.http.",
+    "mozilla."
+  };
+
+  private static final String[] GAE_STANDARD_V1_SHAREED_CLASSES = {
+    "com.google.appengine.api.LifecycleManager",
+    "com.google.apphosting.api.ApiProxy",
```

Comment:
Is this a prefix and will also work for the [inner classes](https://cloud.google.com/appengine/docs/java/javadoc/com/google/apphosting/api/package-summary)?